.github/workflows/release-it.yml

```diff
@@ -0,0 +1,54 @@
+# #########################################################################
+# Creates a new release using `workflow_dispatch` event trigger with `type`
+# as input to describe the type of release to create
+name: 'Release-it: Create a new release on demand'
+
+on:
+  workflow_dispatch:
+    inputs:
+      type:
+        description: 'Type. Can be `patch` `minor` or `major`'
+        required: true
+        default: 'patch'
+
+jobs:
+  release-it:
+    if: github.actor == 'robertsLando'
```

Comment:
Is there a way to use `release-it` without hardcoding a GitHub username?

Reply:
Yeah this is just to prevent someone else to be able to trigger releases, I can add your username if you want or completely remove this check

Reply:
They won't be able to do anything without write access to this repo or the npm package right? If that's the case, I vote to remove it.

Reply:
Yeah, ok!